File: Source/Front-end/front-end/src/app/model/response/registrazione-response.ts
export class RegistrazioneResponse {
  email: string;
  nome: string;
  cognome: string;
  dataDiNascita: Date;

  constructor(email: string, nome: string, cognome: string, dataDiNascita: Date) {
    this.email = email;
    this.nome = nome;
    this.cognome = cognome;
    this.dataDiNascita = dataDiNascita;
  }
}

File: Source/Back-end/src/main/java/org/elis/manutenzione/dto/response/RegistrazioneResponse.java
package org.elis.manutenzione.dto.response;

import lombok.Data;
import java.time.LocalDate;

/**
 * DTO che rappresenta la risposta alla richiesta di registrazione di un utente.
 */
@Data
public class RegistrazioneResponse {

    /**
     * L'indirizzo email dell'utente registrato.
     */
    private String email;

    /**
     * Il nome dell'utente registrato.
     */
    private String nome;

    /**
     * Il cognome dell'utente registrato.
     */
    private String cognome;

    /**
     * La data di nascita dell'utente registrato.
     */
    private LocalDate dataDiNascita;
}